doxygen/dox/DDLBNF200.dox

```diff
@@ -93,10 +93,11 @@ This section contains a brief explanation of the symbols used in the DDL.
               H5T_NATIVE_LONG  | H5T_NATIVE_ULONG  |
               H5T_NATIVE_LLONG | H5T_NATIVE_ULLONG
 
-<float> ::= H5T_IEEE_F16BE     | H5T_IEEE_F16LE     | 
+<float> ::= H5T_IEEE_F16BE     | H5T_IEEE_F16LE     |
             H5T_IEEE_F32BE     | H5T_IEEE_F32LE     |
             H5T_IEEE_F64BE     | H5T_IEEE_F64LE     |
-            H5T_NATIVE_FLOAT16 | H5T_NATIVE_FLOAT   | 
+            H5T_FLT_BF16BE     | H5T_FLT_BF16LE     |
```

Comment:
Open to any alternative names that seem more fitting, but this type is distinct from the IEEE standard, so I basically created a new category for alternative floating-point formats.

Reply:
Perhaps "FLOAT" instead of "FLT"?

Reply:
Seems reasonable to me

Reply:
Agree - I like "FLOAT" better also

Reply:
Actually, maybe "NONSTD", since it's a contrast to the "IEEE" label?

Reply:
So... `H5T_NONSTD_BF16` or `H5T_NONSTD_BFLOAT16`? I like "FLOAT" explicitly.

Reply:
The issue I see with that is that it's not unlikely that these types get adopted into some standard in the future, whether it's IEEE or not.